ql/src/test/results/clientpositive/llap/date_timestamp_partition_filter.q.out

```diff
@@ -0,0 +1,214 @@
+PREHOOK: query: CREATE EXTERNAL TABLE testpd(col1 string, col2 String) PARTITIONED BY(PartitionDate DATE) STORED AS ORC
```

Comment:
Is there a part in this output that shows that partition pruning is working and there are no hidden exceptions that just prevent it from running?

Reply:
https://github.com/apache/hive/pull/4760/files#diff-a03353b02b81f6c2c36cc130d0d85b019fb725c7cb0d57d0baa9b7122177610eR41 and https://github.com/apache/hive/pull/4760/files#diff-a03353b02b81f6c2c36cc130d0d85b019fb725c7cb0d57d0baa9b7122177610eR90

These two lines shows that partition pruning is happening.

Reply:
https://github.com/apache/hive/pull/4760/files#diff-a03353b02b81f6c2c36cc130d0d85b019fb725c7cb0d57d0baa9b7122177610eR41 and https://github.com/apache/hive/pull/4760/files#diff-a03353b02b81f6c2c36cc130d0d85b019fb725c7cb0d57d0baa9b7122177610eR90 shows partition pruning is happening but this is at storage level.

Reply:
Before fix the [output](https://github.com/apache/hive/pull/4760/files#diff-a03353b02b81f6c2c36cc130d0d85b019fb725c7cb0d57d0baa9b7122177610eR107) was empty, after fix it is returning the corresponding partition matching rows.

Reply:
The query result is definitely useful, I am asking for the EXPLAIN output.

Reply:
There is no change in the EXPLAIN plan output, I just added to make sure partition filter pushdown is happening properly.